feature-libs/checkout/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
We now use eslint. We should no longer add tslint files

Reply:
Thanks.  Fixed.
This PR was a bit stale.  I also updated angular.json.